pkg/serverless/proxy/proxy.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build serverlessexperimental
```

Comment:
What is the advantage of having a special build tag for this feature? It seems to me that an env var would be enough to enable/disable the feature. If I'm not mistaken, if we have a separate build tag, we would need to start testing two different builds of the extension.

Reply:
You're right, I've removed the build tag